noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -1014,10 +1463,14 @@
                 ]]>
             </warnings>
         </queries>
-    </rule>
+    </test>
 
-    <rule>
+    <test id="AST10">
         <title>Record count</title>
+        <description>
+            Provides information about the number of Records per Series
```

Comment:
`Provides information about the number of records per series and tests whether the corresponding number specified in arkivuttrekk.xml is correct`